docs/user/glossary.rst

```diff
@@ -6,6 +6,11 @@ so that you have a reference for how we're using them.
 
 .. glossary::
 
+   CI/CD
+      CI/CD is a common way to write *Continuous Integration and Continuous Deployment*.
+      In some scenarios, they exist as two separate platforms.
+      Read the Docs contains a combined CI/CD platform made for documentation.
```

Comment:
```suggestion
      Read the Docs is a combined CI/CD platform made for documentation.
```

Reply:
I wrote contains because we do other things, like flyout menu, CDN, redirects etc... but if you think the _is_ doesn't make us sound like we're _only_ a CI/CD, then I think it's fine :)

Reply:
I think you are right that _contains_ is confusing, and just writing _is_ is better :)